content/docs/ref/publish.md

```diff
@@ -4,7 +4,8 @@
 cml publish [options] <image file>
 ```
 
-Publicly host an image for displaying in a CML report.
+Publicly host an image for displaying in a CML report. Used internally by
```

Comment:
Looks like this needs to be updated.

Reply:
What needs updating?

Reply:
Shouldn't we include a reference to minroud here?